### ISSUE TITLE
Crash during start

### ISSUE THREAD

**Issue by toxygen** [mathstuf/abagames-torus-trooper#1]
Unfortunately, game crashes during launch.

See attached crash report

[tt_2016-03-13-175424_titan.txt](https://github.com/mathstuf/abagames-torus-trooper/files/171105/tt_2016-03-13-175424_titan.txt)

**Comment by mathstuf** [comment]
Something to do with save game load/restore. Do you have a save file already?

**Comment by toxygen** [comment]
No, no save file. 

By any chance, do you also provide compiled binary?

**Comment by mathstuf** [comment]
No, no binaries. I do not have an Apple machine to make OS X binaries anyways. I've only personally tested this one on Linux. I tested gunroar and mu-cade on Windows too, but none on OS X. FYI, this one still does need some work. I'll have to check my local copy for what's laying around.
